torch/testing/_internal/common_utils.py

```diff
@@ -63,38 +63,31 @@
 import numpy as np
 
 import __main__  # type: ignore[import]
+
 import torch
 import torch.backends.cudnn
 import torch.backends.mkl
 import torch.backends.mps
 import torch.backends.xnnpack
 import torch.cuda
+import torch.utils._pytree as pytree
 from torch import Tensor
 from torch._C import ScriptDict, ScriptList  # type: ignore[attr-defined]
 from torch._utils_internal import get_writable_path
-from torch.nn import (
-    ModuleDict,
-    ModuleList,
-    ParameterDict,
-    ParameterList,
-    Sequential,
-)
-from torch.onnx import (
-    register_custom_op_symbolic,
-    unregister_custom_op_symbolic,
-)
+from torch.nn import ModuleDict, ModuleList, ParameterDict, ParameterList, Sequential
```

Comment:
How any of those changes relate to PR in question?

Reply:
Reverted unrelated changes.